### PR TITLE
Respect globs

### PR DESCRIPTION
**Pull request by wkirby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,29 +23,40 @@ And then include the plugin in a file located in your `_plugins` directory. You
...
+    compress_html: true
 ```
 
 ### exclude (default: [])
 
-`exclude` is an array of file paths that will be ignored by `jekyll-tidy`. At the moment the exclusions are driven by simple string matching, which means globs won't match files.
+`exclude` is an array of relative file paths that will be ignored by `jekyll-tidy`.
+
+```
+  exclude: [index.html] # excludes only index.html
```
Comment (ncallaway) [inline README.md]: Should probably add quotes here, if you've added quotes above.

File: README.md [hunk 2]
```diff
@@ -23,29 +23,40 @@ And then include the plugin in a file located in your `_plugins` directory. You
...
+```
+
+`exclude` can also take a glob of file paths. The following, for example, will exclude all `.md` files in the `_posts` directory, but will run `jekyll-tidy` on all other files in `_posts`.
 
 ```
-    exclude: [index.html] # excludes only index.html
-    exclude: ["bin/*"]    # does not exclude everything from the bin/ directory
+  exclude: [_posts/*.md]
```
Comment (ncallaway) [inline README.md]: Also should add quotes around this, and should probably add a comment: `# ignores all markdown files directly within the posts directory.`

I could also see adding another example below this:

`exclude: ["_posts/**/*.md"] # excludes all markdown files anywhere within the _posts directory`

**Comment by ncallaway** [comment]
In my defense, I starting writing that prior comment _before_ you added those tests.

**Comment by wkirby** [comment]
@ncallaway I pace my PR updates to make you look as stupid as possible in a public setting

**Pull request merged**